package.json

```diff
@@ -164,40 +168,40 @@
     "doctoc": "^1.0.0",
     "escape-string-regexp": "^1.0.5 ",
     "esdoc": "^0.4.7",
-    "eslint": "^3.8.0",
+    "eslint": "^4.0.0",
```

Comment:
not relevant to this PR, but I'm feeling we just need to drop eslint and use tslint everywhere..